src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -348,6 +348,10 @@ private AIState waitForNeededItems()
     @NotNull
     private AIState lookForNeededItems()
     {
+        /*
+        * This Function causes "itemsCurrentlyNeeded.isEmpty()= ALWAYS EMPTY" which results in
+        * "if (itemsCurrentlyNeeded.isEmpty()) -> return IDLE" to be alwys true. therefore never checking his chest when actually building
```

Comment:
I suggest instead of commenting the bug that we fix the bug :) If it is always true that is definitely a bug needing to be fixed